src/app/zones/views/ZoneCreateView.vue

```diff
@@ -245,8 +276,21 @@ async function createZone() {
   try {
     zone.value = await kumaApi.createZone({ name: name.value })
   } catch (err) {
-    if (err instanceof Error) {
-      changingError.value = err
+    if (err instanceof ApiError) {
+      errorState.value = {
+        error: err,
+        title: t(`zones.create.statusError.${err.statusCode}.title`, { zoneName: name.value }),
```

Comment:
I chose to put the status codes in the translation files because that should make it easier to find for folks that want to change the messaging around something specific that don't necessarily work on the project.

I can however add a check for handled status code so that a default error is used for status codes that don’t have error messages.

Reply:
Ok as long as we don't have various error codes that need the same text (and hence we'd end up duplicating lots of text). I think at the very least having text for error codes we don't specifically watch out for addresses what I was mostly worried about.